pipeline.sh

```diff
@@ -5,10 +5,15 @@ set -e -o pipefail
 make medius
 make cluster-up
 
+focus="cirros:6.1"
```

Comment:
You can also write this as:

```
focus=${1:-cirros:6.1}
```

But what about as an env variable here instead of an arg?

```
FOCUS=${FOCUS:-cirros:6.1}
```

Reply:
ACK